pkgs/by-name/sl/slurm/package.nix

```diff
@@ -144,16 +145,34 @@ stdenv.mkDerivation rec {
     ]
     ++ (lib.optional enableGtk2 "--disable-gtktest")
     ++ (lib.optional (!enableX11) "--disable-x11")
-    ++ (lib.optional (enableNVML) "--with-nvml");
+    ++ (lib.optional enableNVML "--with-nvml")
+    ++ (lib.optional enablePAM "--enable-pam --with-pam_dir=/build/_out/lib/security");
```

Comment:
It is probably not a good idea to refer to `/build` via an absolute path. `_out/lib/security` should be sufficient?

Reply:
Why do you believe it isn't a good idea? Did you even try your own suggestion? Do note that the slurm configure scripts require the absolute path.

Reply:
It is unfortunately the case that the path has to be absolute

Reply:
We can not assume that `/build` is a stable directory. (1) This directory is configurable via `sandbox-build-dir`. (2) The build fails when the sandbox is turned off via `sandbox false`.
Is there a way to add a patch that allows for a more stable solution?

Reply:
Right, I didn't realize this relied on the sandbox.

Reply:
Swtiched to placeholder "out" instead of build